libs/remixd/src/services/slitherClient.ts

```diff
@@ -140,6 +149,13 @@ export class SlitherClient extends PluginClient {
         if (existsSync(outputFileAbsPath)) {
           let report = readFileSync(outputFileAbsPath, 'utf8')
           report = JSON.parse(report)
+          try {
+            unlink(outputFileAbsPath, (err) => {
```

Comment:
We can also use `unlinkSync`

Reply:
we don't really need to wait for the result.

Reply:
ok